File: server/src/main/java/maker/server/Entity/Weather.java
package maker.server.Entity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Weather {
    private double Probability;
    private double humidity;
    private String state;
    private double present;

}

File: android/app/src/main/java/com/maker/hanger/data/Weather.kt
package com.maker.hanger.data

import com.google.gson.annotations.SerializedName

data class Weather(
    @SerializedName("probability") var probability: Double,
    @SerializedName("humidity") var humidity: Double,
    @SerializedName("state") var state: String,
    @SerializedName("present") var present: Double,
)

data class WeatherResponse(
    @SerializedName("status") var status: Int,
    @SerializedName("message") var message: String,
    @SerializedName("weather") var weather: Weather
)